main/src/main/scala/sbt/Defaults.scala

```diff
@@ -1706,7 +1706,7 @@ object Defaults extends BuildCommon {
         config,
         s.cacheStoreFactory,
         s.log,
-        sys.env.get("SOURCE_DATE_EPOCH").map(_.toLong * 1000).orElse(Some(0L))
+        sys.env.get("SOURCE_DATE_EPOCH").map(_.toLong * 1000)
```

Comment:
Zeroing out the JAR was done so intentionally in https://github.com/sbt/sbt/pull/5344 to make the packaging more reproducible.

Reply:
Ah... Alright I should need to read to understand it but I have a very quick question.
Should it be `0L` or any fixed epoch milliseconds? If it's latter one, can it be `Some(System.currentTimeMillis())`?

Reply:
OK I got it. I will use sbt 1.3.13 or use `SOURCE_DATE_EPOCH` env var if the timestamp of the class file is important. Thanks!